src/ace/managers/system.c

```diff
@@ -300,7 +302,11 @@ void systemCreate(void) {
 	WaitTOF(); // Wait for interlaced screen to finish
 
 	// get VBR location on 68010+ machine
-	// TODO VBR
+	if (SysBase->AttnFlags & AFF_68010)
+	{
+		UWORD getvbr[] = {0x4e7a, 0x0801, 0x4e73};
```

Comment:
change `getvbr` name to something codestyle compliant - p since it's a pointer (array but it's virtually the same), so perhaps something like `pGetVbrCode`?